io-engine/src/grpc/v1/nexus.rs

```diff
@@ -572,6 +574,7 @@ impl NexusRpc for NexusService {
                 .map_err(|_| Status::cancelled("cancelled"))?
                 .map_err(Status::from)
                 .map(|nexus| {
+                    event.generate();
```

Comment:
This should be added inline with the code itself? Otherwise if there's a timeout or disconnect we might miss this?

Reply:
Yes. Will do that.

Reply:
Done.